test.php

```diff
@@ -0,0 +1 @@
+<?php var_dump(\DDtrace\active_span() == true);
```

Comment:
Not used, right? Or am I missing something?

Reply:
You aren't missing anything and it shall be removed :-D